go/cmd/mysql_probe/main.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"context"
+	"net/http"
+	"time"
+
+	"github.com/coreos/go-systemd/daemon"
+	"github.com/gorilla/mux"
+	"github.com/inverse-inc/go-utils/log"
+	"github.com/inverse-inc/packetfence/go/galeraautofix/mariadb"
+)
+
+var ctx = context.Background()
+
+func main() {
+
+	router := mux.NewRouter()
+	router.HandleFunc("/", checkDB).Methods("OPTIONS")
+
+	srv := &http.Server{
+		Addr:        ":3307",
+		IdleTimeout: 5 * time.Second,
+		Handler:     router,
+	}
+	// Systemd
+	daemon.SdNotify(false, "READY=1")
+
+	go func() {
+		interval, err := daemon.SdWatchdogEnabled(false)
+		if err != nil || interval == 0 {
+			return
```

Comment:
I never liked silent returns for errors

Let's make it explicit that there was an error or that the interval was 0 and log the behavior.